esp/src/src-react/components/GetDFSCSV.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react";
+import { DaliService } from "@hpcc-js/comms";
+import { XMLSourceEditor } from "./SourceEditor";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { DefaultButton } from "@fluentui/react";
+
+const logger = scopedLogger("src-react/components/GetDFSCSV.tsx");
+
+const daliService = new DaliService({ baseUrl: "" });
+
+
+interface GetDFSCSVProps {
+
+}
+
+export const GetDFSCSV: React.FunctionComponent<GetDFSCSVProps> = ({
+
+}) => {
+
+    const [dfsCsv, setDfsCsv] = React.useState<string>("");
+    const [logicalNameMask, setLogicalNameMask] = React.useState<string>("");
+    
+    const onSubmit = React.useCallback(() => { daliService.GetDFSCSV({LogicalNameMask: logicalNameMask}).then(( response ) => {
+        setDfsCsv(response.Result);
+    }).catch(err => logger.error(err)); }, [logicalNameMask]);
+
+    return <div>
+        <TableGroup fields={{
+        "LogicalNameMask": { label: nlsHPCC.LogicalNameMask, type: "string" },
```

Comment:
What's missing here is the value getting passed into the field. You could compare to the TableGroup in [WorkunitSummary.tsx](https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/src-react/components/WorkunitSummary.tsx#L179)